martha/martha_v3.js

```diff
@@ -87,6 +88,16 @@ const DG_COMPACT_THE_ANVIL = 'dg.anv0';
 const DG_COMPACT_CRDC = 'dg.4dfc';
 const DG_COMPACT_KIDS_FIRST = 'dg.f82a1a';
 
+// Google Cloud Functions gives us 60 seconds to respond. We'll use most of it to fetch what we can,
+// but not fail if we happen to time out when fetching a signed URL takes too long.
+let pencilsDownTime = 58;
```

Comment:
I'm not in love with this variable name, but I was struggling to think of something more conventional. This metaphor made sense to me. Let me know if you don't think this is clear or if you have other suggestions.